libs/state/docs/concepts-and-best-practices.md

```diff
@@ -0,0 +1,133 @@
+# General
+
+**Component Shell and Folder**
+
+- `@Input` bindings are setter
+- `@Output` bindings are state derivations
+- State is injected over the constructor
+- State is displayed over a pipe in the template
+- UI interaction is implemented over `Subjects`
+- use `*ngrxLet` over `*ngIf`
```

Comment:
use *rxLet or leave it out completely since it isn't released yet :)

Reply:
Will update it.

Also we have to rework this doc in any way.. :D